src/prefect/orion/schemas/states.py

```diff
@@ -141,12 +141,12 @@ def result(self, raise_on_failure: bool = True, fetch: Optional[bool] = None):
 
     def to_state_create(self):
         # Backwards compatibility for `to_state_create`
-        from prefect.orion.schemas import State
+        from prefect.client.schemas import State
 
         warnings.warn(
-            "`to_state_create` is not supported by `prefect.orion.schemas.states.State` "
```

Comment:
Most of the time, users do not call this directly so I shall provide a bit more generic of a message.